cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -1487,6 +1487,215 @@ action.summary_index._name = cyences
+| stats count, max(_time) as Last_Failed_Login, values(LogonError) as LogonError, values(ExtendedProperties) as ExtendedProperties by user, ClientIP \
+| iplocation ClientIP \
+| fillnull Country, Region, City value="-" \
+| eval Location=ClientIP." (".count.") | ".Country." | ".Region." | ".City \
+| stats sum(count) as count, max(Last_Failed_Login) as Last_Failed_Login, list(Location) as Location, values(LogonError) as LogonError, values(ExtendedProperties) as ExtendedProperties by user \
+| sort - count \
+| `cs_human_readable_time_format(Last_Failed_Login)` \
+| `cs_o365_failed_login_due_to_mfs_filter`
+action.summary_index = 1
+action.summary_index._name = cyences
+action.summary_index.category = O365
+action.summary_index.alert_name = O365 - Login failure due to multi factor authentication
+
+[O365 - Login failure outside US due to multi factor authentication]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule =	*/31 * * * *
+description = This alert will show the login failure outside US due to multi factor authentication. \
+\
+Data Collection - Office 365 management activity data (Splunk Add-on for Office 365). 
+dispatch.earliest_time = -55m@s
+dispatch.latest_time = -25m@s
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_o365` sourcetype="o365:management:activity" Workload=AzureActiveDirectory Operation=UserLoginFailed (LogonError="DeviceAuthenticationRequired" OR LogonError="UserStrongAuthClientAuthNRequiredInterrupt") \
+| eval ExtendedProperties=mvzip('ExtendedProperties{}.Name','ExtendedProperties{}.Value'," : ") \
+| stats count, max(_time) as Last_Failed_Login, values(LogonError) as LogonError, values(ExtendedProperties) as ExtendedProperties by user, ClientIP \
+| iplocation ClientIP \
+| where Country!=`cs_home_country` \
+| `confirmiplocation` \
+| fillnull Country, Region, City value="-" \
+| eval Location=ClientIP." (".count.") | ".Country." | ".Region." | ".City \
+| stats sum(count) as count, max(Last_Failed_Login) as Last_Failed_Login, list(Location) as Location, values(LogonError) as LogonError, values(ExtendedProperties) as ExtendedProperties by user \
+| sort - count \
+| `cs_human_readable_time_format(Last_Failed_Login)` \
+| `cs_o365_failed_login_due_to_mfs_outside_country_filter`
+action.summary_index = 1
+action.summary_index._name = cyences
+action.summary_index.category = O365
+action.summary_index.alert_name = O365 - Login failure outside US due to multi factor authentication
+
+[O365 - Login from unknown user]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule =	*/31 * * * *
```

Comment:
This cron, executes every 31 minutes.
`Use 9,39 * * * * and 19,49 * * * *`

Reply:
It will run every 30 minutes. 
![image](https://user-images.githubusercontent.com/25617583/156147588-06588228-e7f7-40e6-a193-3790d0ea07bf.png)

Let me know if we still want to change it

Reply:
Actually, we are using specific cron for alerts on the basis of it's frequency. And below is what we are following for all the alerts:

# Cron Details
# -------------------
# Every 10 minutes
#   - 1, 3, 4, 5, 6 and 8 's multiples
# Every 15 minutes
#   - 2, 17, 32, 47 th minutes
#   - 7, 22, 37, 52 th minutes
# Every 30 minutes
#   - 9, 39th minutes or 19, 49th minutes
# Hourly
#   - 29, 59th minutes
# Daily, Weekly, Monthly, Yearly
#   - At 0th Minute

Reply:
Got it. Made changes.